MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -22,7 +22,7 @@ open class RouteProgress {
         didSet {
             assert(legIndex >= 0 && legIndex < route.legs.endIndex)
```

Comment:
nit:
use `.contains` syntax:
```swift
(0..<route.legs.endIndex).contains(legIndex)
```
?

Reply:
I think it's okay to leave this piece of functionality as it is. Don't see any significant difference.